### PR TITLE
Move general functions from timeline stats to helpers utils

### PR DESCRIPTION
**Pull request by AmitFainholts**
padTo2Digits, getHoursAndMinutes and creatUniqueObjArrayByProp now in helpers for general use.

### CODE REVIEW

File: ui-common/src/helpers/Utils.ts [hunk 1]
```diff
@@ -26,4 +26,20 @@ export class Utils {
 
     return bytes.toFixed(dp) + ' ' + units[u];
   }
+
+static padTo2Digits = (num) => {
```
Comment (leon-up9) [inline ui-common/src/helpers/Utils.ts:30]: only i see weird indentation?

Comment (AmitUp9) [reply ui-common/src/helpers/Utils.ts:30]: i am not sure that I see it

Comment (leon-up9) [reply ui-common/src/helpers/Utils.ts:30]: can we fix it?

Comment (AmitUp9) [reply ui-common/src/helpers/Utils.ts]: I did format document, I hope it fix what you meant

**Review by leon-up9 — APPROVED** [review]

**Pull request merged**